test/sequential/test-watch-mode.mjs

```diff
@@ -174,6 +175,41 @@ describe('watch mode', { concurrency: true, timeout: 60_0000 }, () => {
     });
   });
 
+  it('should watch changes to previously loaded dependencies', async () => {
+    const dependencyContent = 'module.exports = {}';
+    const dependency = createTmpFile(dependencyContent);
+    const relativeDependencyPath = `./${path.basename(dependency)}`;
```

Comment:
does `path.relative(path.basename(dependency))` produces the same result for this line? if yes, can we use it?